modules/olmv1-creating-a-service-account.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/managing-ce.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="olmv1-creating-a-service-account_{context}"]
+= Creating a service account
+
+Unlike {olmv0-first}, {olmv1} does not have permissions to install, update, and manage cluster extensions. Cluster administrators must create a service account and assign the role-based access controls (RBAC) required to install, update, and manage cluster extensions.
+
+[IMPORTANT]
+====
+include::snippets/olmv1-known-issue-service-accounts.adoc[]
+====
+
+.Prerequisites
+
+* Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+
+.Procedure
+
+. Create a service account custom resource (CR), similar to the following example:
```

Comment:
2 small things (being nit-picky here so feel free to ignore):
- service accounts aren't a "custom" resource, they are a part of Kube's core APIs. 
- Not sure about the standards in OCP docs, but I would typically expect a reference to an API to use the associated "kind". (i.e instead of "service account" I would expect something like "ServiceAccount")

Reply:
Thanks, Bryce. I'll remove mention of the CR.

Downstream, when we refer to the specific API, we use `ServiceAccount`, but when we make a general reference, like "create a service account," we use lower case.